application/testing/CMakeLists.txt

```diff
@@ -369,6 +373,11 @@ f3d_test(NAME TestVerboseAnimationWrongAnimationTimeLow DATA BoxAnimated.gltf AR
 # Test exit hotkey
 f3d_test(NAME TestInteractionSimpleExit DATA cow.vtp REGEXP "Interactor has been stopped" INTERACTION NO_BASELINE) #Escape;
 
+if(UNIX AND NOT APPLE AND VTK_VERSION VERSION_GREATER_EQUAL 9.3.20240914)
+  f3d_test(NAME TestInteractionSimpleExitHeadless DATA cow.vtp REGEXP "Interactor has been stopped" INTERACTION NO_BASELINE) #Escape;
+  set_tests_properties(f3d::TestInteractionSimpleExitHeadless PROPERTIES ENVIRONMENT "DISPLAY=")
+endif()
```

Comment:
since you remove the interactor completely, I dont think this test is needed anymore, its basically the same as TestInteractionSimpleExit.